lib/rules/boolean-prop-naming.js

```diff
@@ -22,6 +22,7 @@ const messages = {
   patternMismatch: 'Prop name `{{propName}}` doesn’t match rule `{{pattern}}`',
 };
 
+/** @type { import('eslint').Rule.RuleModule } */
```

Comment:
```suggestion
/** @type {import('eslint').Rule.RuleModule} */
```
please make a similar change throughout

Reply:
sorry😢
Fixed in [94e0e85](https://github.com/jsx-eslint/eslint-plugin-react/pull/3732/commits/94e0e85f6455ee63d2e0311de31fef6fc5f141e8)